src/lib/editor-routes.js

```diff
@@ -0,0 +1,20 @@
+import { Route, Redirect } from 'react-router';
+import React from 'react';
+import AppController from 'components/editor/AppController';
+import ArticleListController from 'components/editor/ArticleListController';
+import ArticleController from 'components/editor/ArticleController';
+import AuthorListController from 'components/editor/AuthorListController';
+import AuthorController from 'components/editor/AuthorController';
+
+export default (
+  <Route path="/" component={AppController}>
+    <Route path="articles/page/:page/" component={ArticleListController}>
+      <Route path="article/slug/:slug/" component={ArticleController} />
+      <Redirect to="article/slug/:slug" from="article/slug/:slug/" />
```

Comment:
Okay cool, thanks for the tip, always good to know the standards. Then I'll change the redirects to go the other way and change that paths correspondingly.